<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Backlight</class>
 <widget class="QMainWindow" name="Backlight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Backlight</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="bl_slider">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>411</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLabel" name="val_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QPushButton" name="onoff_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>On/OFF</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
